keyboard/src/modules/Keyboard.rb

```diff
@@ -556,8 +556,9 @@ def SetKeyboard(keyboard)
         return false # Error
       end
 
-      # Console command...
-      @ckb_cmd = Ops.add("/bin/loadkeys ", @keymap)
+      # Console command. It specifies all tty devices (bsc#1010938)
+      @tty_devices ||= Dir["/dev/tty*"].map { |d| "-C #{d}" }.join(" ")
```

Comment:
@lslezak AFAIK, serial devices are attached to `/dev/ttyS*` devices, so we're covered here. On the other hand, I don't expect `/dev/tty*` devices to not exist.